client/state/i18n/locale-suggestions/reducer.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import {
+	I18N_LOCALE_SUGGESTIONS_REQUEST,
+	I18N_LOCALE_SUGGESTIONS_SUCCESS,
+	I18N_LOCALE_SUGGESTIONS_FAILURE,
+} from 'state/action-types';
+
+import { combineReducers } from 'state/utils';
+
+export const isFetching = ( state = false, action ) => {
+	switch ( action.type ) {
+		case I18N_LOCALE_SUGGESTIONS_REQUEST:
+			return true;
+		case I18N_LOCALE_SUGGESTIONS_SUCCESS:
+		case I18N_LOCALE_SUGGESTIONS_FAILURE:
+			return false;
+		default:
+			return false;
+	}
+};
+
+export const error = ( state = null, action ) => {
```

Comment:
Similarly we can dispatch error events as side effects in data-layer middleware, and generally do not need to store errors in state.